### DISCUSSION TITLE
v0.17-dev

### DISCUSSION THREAD

**Discussion by sriramveeraghanta** [makeplane/plane#4116] [Announcements]
## Features

**Archive Series**

- **Project Archives**— Need to pause a project? Archive it. Now you can focus on what's important. You can view archived projects by applying filters on the Project tab. #4014 
![Project_archiving](https://github.com/makeplane/plane/assets/9498163/03612b75-f1ef-423a-aae7-b8ef0fec9df6)    
- **Cycles and Modules Archives**— Archive your completed Cycles and Modules. Declutter your Cycles and Modules tab for better clarity. You can revisit or restore them later. #3990
![Archiving_C_and_M](https://github.com/makeplane/plane/assets/9498163/a9e97319-7913-4873-a409-6e7052c33f57)
- **Issue Archiving**— You can now manually archive `Completed` or `Cancelled` issues on any layout using the context menu of issues. On issue hover, click the ellipsis to open the issue context menu. #3801 
![Archive_and_restore_issues](https://github.com/makeplane/plane/assets/9498163/9a8f6e41-9087-4bbd-ba6f-1d30bad032eb)

**Face-lift series**

- **Listing pages**— Search and find what you need faster with `Sort`  and `Filter` dropdown in Cycles, Modules and Projects. #3947, #3926, #3910
![Sorting_and_filtering_P_M_and_C](https://github.com/makeplane/plane/assets/9498163/882e127f-85ea-4967-8cda-9c8b11248c1b)
- **Emoji picker for Projects**— The emoji picker for Projects now comes with Categorisation, search and skin tone. #3868
![Enhanced_emoji_picker](https://github.com/makeplane/plane/assets/9498163/25dcb516-4073-422f-91cb-0b9587cebcbc)
- **Customize your projects from the very beginning:** When creating a project, you can now toggle project features. Saves so many clicks! #3909
![Project_feature_selection_while_project_creation](https://github.com/makeplane/plane/assets/9498163/5501b726-6e87-4b63-8e1d-d4fc28db99af)    
- All new alerts-- Less distraction, more responsive, and finally slick. #3836
![Toast alerts](https://github.com/makeplane/plane/assets/9498163/8776df2a-ea6e-4be9-b6c8-18a5c754f12d)

## Improvements

-  The spreadsheet layout is now fully accessible via keyboard. #3833
- After spreadsheet layout, now you can navigate from the issue title to the issue description to properties and so on by using the tab key. Keyboard first approach for the win 💪🏻. #3902 
- You've asked for it, we've fixed it. Peek overview mode is now in full parity with the issue detail full-screen page. You can now view sub-issues and attachments. Closes #2515, #3810, and #3956.
- Minor improvements have been made to enhance the onboarding experience. We've added asterisks (*) for mandatory fields when creating your workspace. #4003
- The side-kicks, or the sidebars for modules and cycles, will now be responsive even with more description. #4000
- When creating issues with `Create more` on, the focus returns to the `Issue title` after an issue is created #3998
- When adding a background color to a table's row or column in the editor, you can also preview how the text content will appear inside it. #3867
- Quickly find progress or blockers in your active cycle with the updated Active Cycle section of your project. #3997
- You can now test your SMTP connection to ensure it works as expected before saving it for everyone in your instance by configuring it. #3988
- Quickly exit peek modes—Close the analytics section by clicking anywhere outside the drawer. #3987
-  Calendar layout now supports responsiveness. #3969
- On mobile screens, we've improved the navigation experience from the sidebar. It now closes with every navigation, or you can manually close it by using the toggle icon on the bottom right. #3960
-  We've removed the `Create issue` action on an empty state of a completed cycle. #3959
-  In the 'Create Issue' modal, the project dropdown will now include projects in which you are participating as an Admin or member. #3952
-  We've removed tooltips on mobile/tablet screens for better user experience. #3948
- We've introduced tooltips for issue reactions. Hovering over an emoji will display the reaction count and show which project members have reacted. #3945
-  Issues in the list layout are now optimized for smooth performance on mobile devices. #3938
-  Issues transferred from a completed cycle now have an empty state screen. #3931
- When you apply quite a few filters, they will be handled gracefully in our UI. #3918
- "Dashboard" is now renamed to "Home" to ensure consistency with the purpose of the page. #3912
- You can now view the issue count on the tab bar. It dynamically updates based on filters and changes across different pages. #3911
- The calendar layout now allows you to assign an existing issue to a specific due date. #3877
- The options you select in the `filter` dropdown will now be displayed at the top of the list. #3869
- The updated 'Home' page now supports filtering by a custom range, allows checking the priorities of assigned issues, and enables viewing all collaborators.. #3849
- Cycle selection dropdown menus have been updated to ensure consistency with the rest of the dropdowns on the platform. #3845
- We’ve adjusted the dimensions of project emoji icons for consistent appearance across the platform. #3837
- The Project UI card has been updated with a fresh new look. #3831
-  The positioning of the dropdown in the spreadsheet layout has now been fixed. #3825
- Closing alerts in the product will no longer close an issue that is open in peek view. #3821
-  Inbox activity and comments now load quickly when an inbox issue is opened. #3819
-  Dropdowns are now fully accessible via keyboard, enhancing efficiency when toggling properties. #3818
-  The comment editor now prevents the accidental addition and saving of empty data as comments. #3817
-  In your SMTP email configuration, you now have the option to use TLS or SSL. #3732
-  Added a 'Get Started' button for easier redirection to `'God Mode'`. #4105

## Bugs

- Issues with saving drafts have been resolved. #3966 
-  Previously, omitting the selection of start or end dates when picking ranges on the 'Home' page resulted in unexpected behavior. This issue has now been fixed. #3934
- We've resolved the tooltip issues in peek-over mode. #3946 
-  We've resolved the scrollbar issue in the List layout, eliminating any overlap. #3933 
-  We've resolved the issues with the filter drop-downs on the dashboard page. #3950 
-  Properties of cross-project sub-issues are now clearly visible in the parent issue. #3996 
-  The labels dropdown in issue detail views has now been fixed. #3995 
-  After leaving a workspace, it will no longer appear in your profile. #3993 
-  Your cache will be invalidated when you leave or delete your workspace. #3989 
-  Dates within the product are now consistent across different timezones, addressing the issues #3074, #3305, #3903, and #3986.
-  The headers in cycle and module peek views are now fixed. #3953 
-  When deleting or archiving an issue in peek overview mode, the peek view will now remain open. #3944 
-  Gantt layout for modules and cycles now displays a pointer cursor when you hover over a module or cycle. #3940 
- Module and cycle peek views are now scrollable, allowing visibility of all assignees, labels, and states. #3932 
- Code blocks will no longer automatically convert to inline code blocks when transitioning to the read-only mode of the editor. #3925 
- Creating a new issue from a module without any existing issues was not functioning as expected. This issue has been resolved. #3916 
-  Child issues will not be visible when adding a parent to an issue. #3914 
-  Date-range pickers now function correctly when selecting dates. #3913 
- You can now deselect project leads if they are accidentally selected during project creation. #3898 
-  The jitter issues with project lead or default assignee selections have been resolved. #3894 
-  The cursor position inconsistency of the divider block in pages has been resolved to meet expectations. #3890 
- Attempting to add issues to a completed cycle will now display an error alert. #3889 
- Using an AI response will now add the response only at the location of your cursor. #3872 
- Longer state names are now properly handled in the Kanban view. #3866 
- Links now support bold and italic formatting in the editors. #3855 
- The URLs within issue links now support more than 200 characters. #3852 
- Module descriptions are now functioning as expected in the sidebar. #3844 
-  Projects you have favorited, even if you are not a part of them, will no longer appear in your side nav. #3842 
-  Navigation across projects on smaller screens has been fixed.#3829
- Pressing 'esc' while adding a description to your issue will no longer close the issue's peek view. #3888 
- After adding tables or images in the editor, you can smoothly navigate to the next block. #3856

<hr /><em>This discussion was created from the release <a href='https://github.com/makeplane/plane/releases/tag/v0.17-dev'>v0.17-dev</a>.</em>

**Comment by ovvesley** [comment]
> Dates within the product are now consistent across different timezones, addressing the issues [bug]: Date Input in Gantt is off by one day.  #3074, https://github.com/makeplane/plane/issues/3305, https://github.com/makeplane/plane/issues/3903, and https://github.com/makeplane/plane/pull/3986.

I was eagerly waiting for this fix. Thank you guys!
